src/unit_tests/os_crypto/hmac/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2015, Wazuh Inc.
+#
+# This program is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation.
+
+cmake_minimum_required(VERSION 3.10)
+
+list(APPEND hmac_tests_names "test_hmac_op")
+list(APPEND hmac_tests_flags "-Wl,--wrap,fopen -Wl,--wrap,fclose -Wl,--wrap,fflush -Wl,--wrap,fread \
+                             -Wl,--wrap,fseek -Wl,--wrap,fwrite -Wl,--wrap,remove -Wl,--wrap,fprintf \
+                             -Wl,--wrap,fgets -Wl,--wrap,fgetpos -Wl,--wrap,fgetc -Wl,--wrap,popen")
```

Comment:
The wrap for function "popen" is not needed in this test, since it is not used by the called functions in it.

Reply:
Yes, it is necessary because the `__wrap_popen` mock was modified to use test_mode, and it must be added so that it finds `__real_popen`, although it is not used directly, if there are `popen` calls in the test

Reply:
Ditto

Reply:
As discussed in the huddle, it was determined that they were necessary.